docs/about/join.md

```diff
@@ -71,17 +84,20 @@ To apply for an account on the Wynton HPC cluster, please fill out the following
 
  <div class="form-group">  
   <label for="ucsf_id">UCSF Employee/Student ID</label> <span style="color: red">*</span><br>
-  <input type="text" id="ucsf_id" name="ucsf_id" placeholder="02xxxxxxx or 'none'" pattern="(\d{9}|none)" maxlength="9" size="15" required> <small>(nine digits; to find yours, go to <a href="https://myaccess.ucsf.edu/myid">https://myaccess.ucsf.edu/myid</a>)</small><br>
+  <input type="text" id="ucsf_id" name="ucsf_id" placeholder="02xxxxxxx or 'none'" pattern="(\d{9}|none)" maxlength="9" size="15" required><br> 
+  <small>(**Nine digits: To find yours, go to <a href="https://myaccess.ucsf.edu/myid">https://myaccess.ucsf.edu/myid</a>)</small><br>
```

Comment:
What's the purpose of `**`?  Is that meant to be Markdown code for `<strong>` or maybe `<em>`?

Reply:
made changes as per suggestion